gtsam/nonlinear/tests/testCallRecord.cpp

```diff
@@ -98,7 +98,8 @@ struct Record: public internal::CallRecordImplementor<Record, Cols> {
   friend struct internal::CallRecordImplementor;
 };
 
-internal::JacobianMap & NJM= *static_cast<internal::JacobianMap *>(nullptr);
+internal::JacobianMap* NJM_ptr = static_cast<internal::JacobianMap *>(nullptr);
+internal::JacobianMap & NJM = *NJM_ptr;
```

Comment:
What was the warning here?

Reply:
I guess the direct deferrencing of a nullptr...  it's ingenious that way of obtaining a *reference* to nullptr, btw :-)

Reply:
I know lol. I was cracking up when I saw that this workaround didn't produce the error.